server/models/userAccessObjectModel.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const {Schema} = mongoose;
+
+const userAccessSchema = new Schema({
+  AccessLevel: String,
+  robotId: mongoose.Types.ObjectId,
+  userId: mongoose.Types.ObjectId
+}, { collection: 'completeCollection' });
```

Comment:
Can you explain this to me? In the video tutorial stated in the wiki, the instructor created a user model and a note model. Shouldn't we create a user model and then create an entry in the ssot to link it?

Reply:
I actually didn't yet create a user schema, as I think it schould be done in the US to introduce user management.
To be able to test our SSoT saving, I ceated this object/schema, to join a SSoT and a user and giving it the additional attribute of the accesslevel, so we could start using that sooner with the hardcoded value of the user Id in our Code